python/python/bystro/proteomics/tests/test_proteomics_types.py

```diff
@@ -11,15 +10,6 @@
 def test_ProteomicsSubmission():
     ProteomicsSubmission("foo.tsv")
 
-
-def test_ProteomicsSubmission_bad_input():
```

Comment:
This test isn't needed because mypy type rules correctly disallow ProteomicsSubmission(3) (why Pat had to provide type: ignore on that line). 

These types are entirely used for internal response processing, so not user facing. Also, in general we do not want to rely on runtime error checking in Python code.